lib/gollum/templates/overview.mustache

```diff
@@ -0,0 +1,42 @@
+<div id="wiki-wrapper" class="results">
+<div id="head" style="border:none;">
```

Comment:
Prefer a non-inline CSS option to keep the templates clean, but no dealbreaker if it's too complicated!

Reply:
Fixed.